terraform/deploy/app/modules.tf

```diff
@@ -33,6 +33,8 @@ module "emr" {
   vpc                        = data.terraform_remote_state.aws_analytical_environment_infra.outputs.vpc
   env_certificate_bucket     = local.env_certificate_bucket
   emp_dir_path               = var.emp_dir_path
-  dataset_s3_paths           = []
   emrfs_kms_key_arns         = []
```

Comment:
This should also be filled with the CMK of the bucket, it's also an output of the ADG terraform